test-utils/src/lib.rs

```diff
@@ -17,6 +17,12 @@
 
 //! Test utils
 
+#[doc(hidden)]
+pub use futures;
+pub use substrate_test_utils_derive::test;
```

Comment:
There are no docs how it works etc.

And pleas also mention that `tokio` is required in the `Cargo.toml` deps.

Reply:
I'm very disappointed because `#[cfg(test)]` seems to be false during doc tests... so I can write an example but I can't really test it or at least not automatically...

So I added a doc with a big warning about the tokio dependency but I can't add a concrete example with running a node since it would be unmaintained.